lib/open_project/configuration.rb

```diff
@@ -175,7 +175,9 @@ module Configuration
       'sentry_dsn' => nil,
       # Allow separate error reporting for frontend errors
       'sentry_frontend_dsn' => nil,
-      'sentry_host' => 'https://sentry.openproject.com',
+      'sentry_host' => 'https://sentry2.openproject.com',
```

Comment:
I think it would be better to have no value as the default here instead of the one very specific to our SaaS setup. This means we have to set them but otherwise instances not belonging to us are more prone to falsely sent requests to our sentry.

Reply:
No requests will be sent unless a `DSN` is configured so this will not happen

Reply:
Sure, but somebody only activating it partly by setting a `sentry_dsn` will trigger it. And having URLs to our SaaS infrastructure in the public code looks off IMO.

Reply:
It will not be triggered since the sentry host will reject invalid DSNs, but I'm fine with removing it if @machisuji can put the respective `OPENPROJECT__SENTRY_HOST` envs for saas.